src/mavoscript.js

```diff
@@ -491,6 +491,14 @@ var _ = Mavo.Script = {
 			},
 			precedence: 2
 		},
+		"groupby": {
+			symbol: "by",
+			code: (array, key) => {
+				Mavo.warn("groupby() is deprecated and will be removed in the next version of Mavo. Please use group_by() instead.");
+				return Mavo.Functions['group_by'](array, key);
```

Comment:
```suggestion
				return Mavo.Functions["group_by"](array, key);
```

To be consistent with the rest of the codebase.